File: C/1036.c
#include <stdio.h>
#include <math.h>
int main()
{
    double a,b,c,d,x,y;
    scanf("%lf %lf %lf",&a,&b,&c);
    d=pow(b,2)-4*a*c;
    if(a==0 || d < 0)
    {
        printf("Impossivel calcular\n");
        return 0;
    }
    else
    {
        x=0.5*(-b+sqrt(d))/a;
        y=0.5*(-b-sqrt(d))/a;
        printf("R1 = %.5lf\n",x);
        printf("R2 = %.5lf\n",y);
    }
    return 0;
}

File: Python/1036.py
import math
a,b,c=[float(x) for x in input().split()]
d=pow(b,2)-4*a*c
if a==0 or d<0:
    print("Impossivel calcular")
else:
    x=0.5*(-b+math.sqrt(d))/a
    y=0.5*(-b-math.sqrt(d))/a
    print(f"R1 = {x:.5f}\nR2 = {y:.5f}")